.github/workflows/test-integrations.yml

```diff
@@ -624,7 +624,7 @@ jobs:
         run: chmod +x consul
       - name: Build consul:local image
         run: docker build -t ${{ env.CONSUL_LATEST_IMAGE_NAME }}:local -f ./build-support/docker/Consul-Dev.dockerfile .
-      - name: Peering commonTopo Integration Tests
+      - name: Peering Integration Tests
```

Comment:
A simple `Peering Integration Tests` implies this is *all* the peering integration tests. There are more in `consul-container`.